www/index.html

```diff
@@ -487,10 +491,17 @@ <h2 data-i18n="configure-title">Configuration</h2>
                             <br /> Scanning for archives... Please wait <img src="img/spinner.gif" alt="Please wait..." />
                         </div>
                         <div id="chooseArchiveFromLocalStorage" style="display: none;">
-                            <br /> Please select the archive you want to use : <select id="archiveList"
-                                class="form-control"></select>
-                            <br /><button type="button" class="btn btn-light" id="btnRescanDeviceStorage">Rescan</button>
-                            Rescans your SD Cards and internal memory
+                            <div style="float: left;display: none;" id="fileCountDisplay">
+                                <p id="numberOfFilesDisplay" style="float: left;">0</p>
+                                <span style="float: left;">&nbsp;archives found in selected location.&nbsp;</span>
+                            </div>
+                            <p style="float: left;">Please select the archive you want to use : </p>
+                            <select id="archiveList" class="form-control"></select>
+                            <br />
+                            <span id="rescanButtonAndText" style="display: none;">
```

Comment:
`<span>` is normally only used inside block elements. This seems like it should be a `<p>` not a `<span>`.